neurodamus/data/hoc/RNGSettings.hoc

```diff
@@ -17,15 +17,13 @@ globalSeed = 0
 begintemplate RNGSettings
 
 public init, interpret, getRNGMode, getIonChannelSeed, getSynapseSeed, getStimulusSeed, getMinisSeed, getGlobalSeed
-public COMPATIBILITY, RANDOM123, UPMCELLRAN4
+public RANDOM123
 
 external rngMode, ionchannelSeed, synapseSeed, stimulusSeed, minisSeed, globalSeed, terminate
 
 proc init() {
     // consts for random number handling
-    COMPATIBILITY = 0
     RANDOM123 = 1
```

Comment:
Can we also remove `rngMode` and `getRNGMode` ?  I see that you have removed them in`connection.py` and `stimulus_manager.py`.

Reply:
OK, apparently, their usages were removed in neurodamus-models, so I can do that.